README.md

```diff
@@ -21,9 +21,9 @@ Check the [releases page](https://github.com/IDPF/epubcheck/releases) to get the
 
 Documentation on how to **use** EpubCheck, to **contribute** to the project or to **translate** messages is available on the [EpubCheck wiki](https://github.com/IDPF/epubcheck/wiki).
 
-Technical discussions are hosted are held on our public [email list](mailto:'public-epubcheck@w3.org'). To subscribe to the email list, send an email with subject "subscribe" to [public-epubcheck@w3.org](mailto:'public-epubcheck@w3.org'). To participate in the discussion, simply send an email to [public-epubcheck@w3.org](mailto:'public-epubcheck@w3.org'). 
+Technical discussions are hosted are held on our public [email list](mailto:public-epubcheck-request@w3.org). To subscribe to the email list, send an email with subject "subscribe" to [public-epubcheck-request@w3.org](mailto:public-epubcheck-request@w3.org). To participate in the discussion, simply send an email to [public-epubcheck-request@w3.org](mailto:public-epubcheck-request@w3.org). 
 
-Historical archives of disussions prior to October 2017 are stored at [EpubCheck Google Group](https://groups.google.com/forum/#!forum/epubcheck).
+Historical archives of discussions prior to October 2017 are stored at [EpubCheck Google Group](https://lists.w3.org/Archives/Public/public-epubcheck/). An archive of the email list beginning in October 2017 https://lists.w3.org/Archives/Public/public-epubcheck/.
```

Comment:
The -request address should only be on subscribing. What about putting the archive first, like this:

Technical discussions are hosted are held on our public [email list](https://lists.w3.org/Archives/Public/public-epubcheck/). To subscribe to the email list, send an email with the subject "subscribe" to [public-epubcheck-request@w3.org](mailto:public-epubcheck-request@w3.org?subject=subscribe). After signing up, simply send emails to [public-epubcheck@w3.org](mailto:public-epubcheck@w3.org) to participate in discussions.

Historical archives of discussions prior to October 2017 are stored at [EpubCheck Google Group](https://lists.w3.org/Archives/Public/public-epubcheck/).

Reply:
fine with me. I should stop editing before coffee.